src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/integration/cas3/Cas3ReportsTest.kt

```diff
@@ -161,6 +163,49 @@ class Cas3ReportsTest : IntegrationTestBase() {
         .isBadRequest
         .expectBody()
         .jsonPath("$.detail").isEqualTo("Start Date $startDate cannot be after End Date $endDate")
+        .jsonPath("invalid-params[0].errorType").isEqualTo("afterEndDate")
+        .jsonPath("invalid-params[0].propertyName").isEqualTo("\$.startDate")
+    }
+  }
+
+  @ParameterizedTest
+  @EnumSource(value = Cas3ReportType::class)
+  fun `Get report returns 400 if start date is the same as end date`(reportType: Cas3ReportType) {
+    `Given a User`(roles = listOf(CAS3_ASSESSOR)) { user, jwt ->
+      val startDate = "2023-08-02"
+      val endDate = "2023-08-02"
+      webTestClient.get()
+        .uri("/cas3/reports/$reportType?startDate=$startDate&endDate=$endDate&probationRegionId=${user.probationRegion.id}")
+        .header("Authorization", "Bearer $jwt")
+        .header("X-Service-Name", ServiceName.temporaryAccommodation.value)
+        .exchange()
+        .expectStatus()
+        .isBadRequest
+        .expectBody()
+        .jsonPath("$.detail").isEqualTo("Start Date $startDate cannot be after End Date $endDate")
```

Comment:
Need to update the message to show that the start date is same as end date

Reply:
I'd actually expect the very same error as the previous test on that test:

```suggestion
        .jsonPath("invalid-params[0].errorType").isEqualTo("afterEndDate")
        .jsonPath("invalid-params[0].propertyName").isEqualTo("\$.startDate")
```